assets/components/Validator/index.js

```diff
@@ -242,7 +242,7 @@ const xValidate = (state) => ({
     setData(data) {
         this.status = data.status;
         this.message = data.message;
-        if (this.onCheck) this.onCheck('error' !== data.status);
+        if (this.onCheck) this.onCheck(!['error', 'loading'].includes(data.status));
```

Comment:
on peut déplacer le check dans le if , wdyt ?

```suggestion
        if (this.onCheck && !['error', 'loading'].includes(data.status)) {
            this.onCheck(true);
        }
```